packages/mermaid/src/docs/.vitepress/theme/index.ts

```diff
@@ -23,7 +23,6 @@ export default {
     return h(Theme.Layout, null, {
       // Keeping this as comment as it took a lot of time to figure out how to add a component to the top bar.
       'home-hero-before': () => h(TopBar),
-      'home-hero-info-before': () => h(ProductHuntBadge),
```

Comment:
Can you remove the ProductHuntBadge file as well? Or is it used somewhere else?

Reply:
We can leave it for future PH launches.